assets/sidecar/developer-hub-app-config-patches.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: v1
+data:
+  RHDH_TOKEN: base64-encoded-token-generated-from-openssl-or-similar-program
+kind: Secret
+metadata:
+  name: ai-rh-developer-hub-env
+  namespace: ai-rhdh
+type: Opaque
+
+---
+apiVersion: v1
+data:
+  app-config.extra.yaml: |-
+    auth:
+      environment: production
+      providers:
+        github:
+          production:
+            clientId: ${GITHUB__APP__CLIENT__ID}
+            clientSecret: ${GITHUB__APP__CLIENT__SECRET}
+    # start of delta from ai-rhdh-installer
+    backend:
+      auth:
+        externalAccess:
+          - type: static
+            options:
+              token: ${RHDH_TOKEN}
+              subject: admin-curl-access
+      reading:
+        allow:
+          - host: example.com
+          - host: '*.mozilla.org'
+          - host: '*.openshift.com'
+          - host: '*.openshiftapps.com'
+          - host: '10.128.*'
+          - host: '127.0.0.1:9090'
+          - host: '127.0.0.1:8080'
+          - host: '127.0.0.1:7070'
+          - host: 'localhost:9090'
+          - host: 'localhost:8080'
+          - host: 'localhost:7070'
+    # end of delta from ai-rhdh-installer
```

Comment:
start of the changes I patch in to this configmap created by the ai-rhdh-installer ... I left the other parts for context